features/auth/locales/en.json

```diff
@@ -82,15 +82,15 @@
     "tos_accept_label": "I Accept the Terms of Service."
   },
   "basic_form": {
-    "name": {
-      "field_label": "Your name",
-      "empty_error": "Please enter your name",
-      "required_error": "This field can't be empty."
-    },
     "email": {
       "field_label": "Email",
       "empty_error": "Please enter a valid email address",
       "required_error": "Email can't be empty."
+    },
+    "password": {
+      "field_label": "Password",
+      "empty_error": "Please enter a password",
```

Comment:
I think this should either mirror the email `empty_error` message by saying "Please enter a valid password" or say something more specific about what made the password invalid, as this is a bit misleading at the moment to tell the user to enter a password when they have done so...